main.py

```diff
@@ -1,7 +1,9 @@
 from aiohttp import web
 from routes import setup_routes
+from aiohttp_swagger import *
+
 
 app = web.Application()
 setup_routes(app)
+setup_swagger(app,swagger_from_file="swagger.yaml")
```

Comment:
Since the default page doesn't seem to be useful. I'd suggest to mount swagger as /.

BTW who is Artem?
<img width="502" alt="Screenshot 2020-12-29 at 10 24 48" src="https://user-images.githubusercontent.com/7992612/103266201-20563080-49c0-11eb-84fb-a592f3dba74f.png">

Reply:
+ update the README correspondingly: curl snippets can be removed since they're available in swagger.

Reply:
Artem did 7,8,9 video courses on YouTube